docs/configure.md

```diff
@@ -471,7 +471,6 @@ If you want to specify a custom Redis server, you'll need specify the correspond
 
 - `sourcegraph-frontend`
 - `repo-updater`
-- `lsif-server`
```

Comment:
Does it no longer use Redis at all? Or do we need to update the instructions here?

Reply:
It does not.